src/Controls/src/Core/Element/Element.cs

```diff
@@ -386,9 +396,9 @@ public Element Parent
 
 		void SetParent(Element value)
 		{
-			Element realParent = RealParent;
+			Element currentParent = Parent;
```

Comment:
Retrieving the Parent here shouldn't throw a warning, we want to check here if the Parent has been collected and if so we don't run the logic that's going to a log a warning